tools/release/download_buildkite_artifacts.py

```diff
@@ -0,0 +1,119 @@
+#
```

Comment:
it's generally best to start python scrpts with a `#!` line:

```suggestion
#!/usr/bin/env python
#
```